index.html

```diff
@@ -51,6 +51,200 @@ <h2>
+                        </ul>
+                    </div>
+                    <p style="padding-bottom: 78px;">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="languages">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div style="padding-top: 24px; padding-bottom: 20px;">
+                        <button>See project</button>
+                    </div>
+
+               </div>
+                
+            </div>
+
+        </section>
+        <section class="about-myself">
+            <div>
+                <h1>
+                    About Myself
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="socials">
+                <h4>
+                    LET’S CONNECT
+                </h4>
+                <ul>
+                    <li>
+                        <a style="width:19.99px; height:19.96px" href="#"><img src="images/Twitter.svg"></a>
+                    </li>
+                    <li>
+                        <a style="width:20px; height:20px" href="#"><img src="images/Linkedin icon.svg"></a>
+                    </li>
+                    <li>
+                        <a style="width:20px; height:16px" href="#"><img src="images/Medium.svg"></a>
+                    </li>
+                    <li>
+                        <a style="width:17.99px; height:20px" href="#"><img src="images/Github.svg"></a>
+                    </li>
+                    <li>
+                        <a style="width:14.17px; height:20px" href="#"><img src="images/Vector.svg"></a>
+                    </li>
+                </ul>
+            </div>
+            <div style="padding-top: 34px;">
+                <button>Get my resume</button>
+            </div>
+            <div class="title">
+                <h3>
+                    Languages
+                </h3>
+                <img src="images/Arrow.svg">
+            </div>
+            <div class="lightweight">
+                <div class="vivo"><img style="padding-left: 12px; padding-right: 16px; padding-top: 12px; padding-bottom: 12px;" src="images/Javascript.svg"><h5>Javascript</h5></div>
+                <div class="vivo"><img style="padding-left: 12px; padding-right: 16px; padding-top: 12px; padding-bottom: 12px;" src="images/HTML.svg"><h5>HTML</h5></div>
+                <div class="vivo"><img style="padding-left: 12px; padding-right: 16px; padding-top: 12px; padding-bottom: 12px;" src="images/CSS.svg"><h5>CSS</h5></div>
```

Comment:
- Kindly, use the external style CSS as your precious code reviewer suggested 👌👌 Always use the class attribute for multiple elements (do not use ID selector for multiple elements). 👍 👍